agent/src/beerocks/slave/son_slave_thread.cpp

```diff
@@ -4744,6 +4746,15 @@ bool slave_thread::handle_ap_capability_query(Socket *sd, ieee1905_1::CmduMessag
     return true;
 }
 
+bool slave_thread::handle_ack_request(Socket *sd, ieee1905_1::CmduMessageRx &cmdu_rx)
```

Comment:
I'm ambivalent about this one... As per #273 there are two options for handling acks: either in the transport, or in each component. However, if acks are handled in the component, they will need to be handled both in the backhaul manager (for messages that are emitted from there) and in the slaves - and the backhaul manager should only forward the acks that do not correspond to a message it sent itself.

I guess for now, however, it's OK to handle the acks in both slaves.